contentcuration/contentcuration/models.py

```diff
@@ -1720,6 +1720,78 @@ class File(models.Model):
 
     objects = CustomManager()
 
+    _edit_filter = Q()
+    for tree_name in CHANNEL_TREES:
+        _edit_filter |= Q(
+            **{
+                "editable_channels__{}__tree_id".format(tree_name): OuterRef(
```

Comment:
What if we coalesced the values? (disclaimer, not entirely sure Django could handle this correctly)
```python
_edit_filter |= Q(
    **{
        "editable_channels__{}__tree_id".format(tree_name): Coalesce(
            OuterRef("contentnode__tree_id"),
            OuterRef("assessment_item__contentnode__tree_id")
        )
    }
)
```

Reply:
Alright so this doesn't work.